gcc/rust/resolve/rust-ast-resolve-unused.h

```diff
@@ -30,7 +30,14 @@ class ScanUnused
   static void ScanRib (Rib *r)
   {
     r->iterate_decls ([&] (NodeId decl_node_id, Location locus) -> bool {
-      if (!r->have_references_for_node (decl_node_id))
+      CanonicalPath ident = CanonicalPath::create_empty ();
+      if (!r->lookup_id (decl_node_id, &ident))
+	{
+	  return false;
```

Comment:
Can you put an assertion that this returns ok? Like:

```
bool ok = r->lookup_id();
rust_assert(ok);
```

 If this breaks the make check-rust then leave it as is.